Surface_mesh_segmentation/examples/Surface_mesh_segmentation/sdf_values_example.cpp

```diff
@@ -17,7 +17,7 @@ int main()
   Polyhedron mesh;
   std::ifstream input("data/cactus.off");
   if ( !input || !(input >> mesh) || mesh.empty() || ( !CGAL::is_triangle_mesh(mesh)) ) {
-    std::cerr << "Input is not a triangle mesh" << std::endl;
+    std::cout << "Input is not a triangle mesh" << std::endl;
```

Comment:
I have never said that `cout` could be used for error output. As far as I remember, your question was about informative messages: sometimes they are displayed to `cout` and sometimes to `cerr`. In my opinion that depends if the informative messages should be considered as the normal output of the program.

That is important that error and debugging messages are displayed to `cerr`, because `cerr` [is always flushed by default][doc cerr] (its flag [`unitbuf`] is set). In case the program does not exit correctly, the outputs to non-flushed buffers may not be displayed completely.

  [doc cerr]: http://en.cppreference.com/w/cpp/io/cerr
  [`unitbuf`]: http://en.cppreference.com/w/cpp/io/ios_base/flags